libyul/backends/evm/EVMDialect.cpp

```diff
@@ -142,6 +142,23 @@ map<YulString, BuiltinFunctionForEVM> createBuiltins(langutil::EVMVersion _evmVe
 			Expression const& arg = _call.arguments.front();
 			_assembly.appendLinkerSymbol(std::get<Literal>(arg).value.str());
 		}));
+
+		builtins.emplace(createFunction(
```

Comment:
Nit: I'd move this to be first or last.

Nit 2: linkersymbol should have come after the `data*` and before setimmutable :)

Reply:
Does this *really* matter :-D? But ok, I'll move both linkersymbol and this to after ``data*``...

Reply:
Actually, I won't... even the ``data*`` things are not ordered... we can order it alphabetically separately, I don't want to clutter this PR even more.

Reply:
(the reason for it being where it is is again a rebase by the way)